.github/CONTRIBUTING.md

```diff
@@ -91,8 +91,8 @@ Before using a local provider version, you'll need to set up [0.13+'s expected d
 You can set up the directories required with the following commands. You should substitute your system's architecture or a different path if desired:
 
 ```
-mkdir -p ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google/5.0.0/darwin_amd64
-mkdir -p ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google-beta/5.0.0/darwin_amd64
+mkdir -p ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google/5.0.0/$(go env GO_HOSTOS)_$(go env GOHOSTARCH)
```

Comment:
Can we leave the example `darwin_amd64` and offer the `go env` method as an option? potentially in the sentence above where we explain substituting the system's architecture